addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -412,7 +421,7 @@ class MessageList extends Component {
             this.threadView.markComponentHintProcessed(hint);
             return;
         }
-        await this._scrollToMessage(message.id);
+        await this._scrollToMostRecentMessage();
```

Comment:
if this needs to be changed, all the conditions above in the method about "message" can probably be removed